HW_4/task_3.py

```diff
@@ -0,0 +1,51 @@
+"""
+Задать натуральное число k.
+Сформируйте многочлен (полином) степени k со случайными коэффициентами из промежутка от 0 до 100, включительно.
+Многочлен вывести в консоль и записать в файл.
+
+Ввод: значение типа <int>
+Вывод: значение типа <str>, файл с одной строкой.
+
+Пример:
+2
+2x^2 + 4x + 5 = 0
+"""
+
+from random import randrange
+from os import path, mkdir, chdir
+
+k = int(input("Введите натуральное число k "))
+polynom = ([randrange(101) for i in range(k + 1)])
```

Comment:
Тут внешние скобки не нужны.
```suggestion
polynom = [randrange(101) for i in range(k + 1)]
```